### ISSUE TITLE
Add reticulate/virtualenv advice to short vignette

### ISSUE THREAD

**Issue by pbulsink** [SCasanova/f1dataR#119]
You can cause reticulate to use a virtual environment by running (change the first argument to name the venv whatever you want): 
```
reticulate::virtualenv_create('f1dataRenv', packages = 'fastf1')
reticulate::use_virtualenv('f1dataRenv')
```
You might need to restart the R session to get reticulate to use the venv. If so, run that second line *before* calling library(fastf1)

_Originally posted by @pbulsink in https://github.com/SCasanova/f1dataR/issues/39#issuecomment-1528320699_

**Comment by SCasanova** [comment]
Is this something we should include during package load?

**Comment by pbulsink** [comment]
I'm not sure. I have a Windows machine that I'm doing some tests on, this
came up on that device so it might be more common than just the one bug
report.

I think it might be bad practice to change other packages behavior on
behalf of the user (i.e. by setting the `reticulate` behavior), because the
user might have a specific virtualenv, might want to use conda, or just the
default system python.

So I think a vignette/docs with this info and a pointer to that in the
readme is likely sufficient.

**Comment by pbulsink** [comment]
I think maybe a helper function could do it on the request of the user, but not automatically overwrite an advanced users configuration. 

Something like (pseudo code):
```
setup_fastf1 <- function() {
  # check for standard 'f1dataRvenv' virtualenv or condaenv and activate
  if ('f1dataRvenv' in reticulate::list_venv()) {
    reticulate::use_venv('f1dataRvenv')
  } else if ('f1dataRcondaenv') {
    # etc...
  } else {
    # make new venv
    reticulate::create_venv('f1dataRvenv', packages = c('numpy', 'fastf1'))
}
```

It could possibly also take a venv/condaenv name to use instead, as an argument.

**Comment by SCasanova** [comment]
Mmm yes I like that. I also recall having to do something similar at some point but I thought nothing of it back then

**Comment by pbulsink** [comment]
I'll have a go at this hopefully before the weekend. I'd consider this as blocking a submission to CRAN

**Comment by SCasanova** [comment]
Ok sounds good. Maybe after this we submit?

**Comment by pbulsink** [comment]
Closed by #126
